### PR TITLE
Change method of running the two bots, add command that prints queue

### LINKED ISSUE #47: !squeue or !sq - Print out the current queue.

### PR DESCRIPTION
**Pull request by notdisliked**
resolves #47

### CODE REVIEW

File: cogs/sounds.py [hunk 1]
```diff
@@ -58,8 +63,10 @@ def __init__(self, bot : commands.Bot):
             ydl = youtube_dl.YoutubeDL(opts)
             func = functools.partial(ydl.extract_info, source, download=False)
             info = await self.bot.loop.run_in_executor(None, func)
+            print(info)
```
Comment (tinlun123) [inline cogs/sounds.py]: Print statement pls.

File: BotGeBeO.py [hunk 2]
```diff
@@ -35,5 +35,8 @@ class BotGeBeO(commands.Bot):
         f = open("tokens/helpers.cfg")
         helper_tokens = f.readlines()
         helper_tokens = [x.strip() for x in helper_tokens]
+        print(helper_tokens)
```
Comment (tinlun123) [inline BotGeBeO.py]: Hey cmon

File: helpers/helper_bot.py [hunk 3]
```diff
@@ -8,25 +8,15 @@ class HelperBot(commands.Bot):
     """ 
     Helper bot that gets spawned in a different thread to help the main bot with various things.
     The general pattern for the class is:
-    - Any async functions should prefixed with _ and called using asyncio.run_coroutine_threadsafe.
-    - asyncio.run_coroutine_threadsafe spawns a new thread that allows async calls to be non-blocking.
-    - Async functions (i.e. any _ prefix methods) should NOT be called externally.
     - Variables should not be modified externally. Variables should be modified by methods only. (it's OOP)
```
Comment (tinlun123) [inline helpers/helper_bot.py]: Probably reword this since there's no bullet points anymore.

**Comment by heartles** [comment]
Tin's review changes incorporated, dismissing them and merging.

**Pull request merged**